src/cache_cf.cc

```diff
@@ -3918,6 +3920,20 @@ configFreeMemory(void)
 #if USE_OPENSSL
     Ssl::unloadSquidUntrusted();
 #endif
+    Config.lifecycleEnd();
+}
+
+void
+SquidConfig::lifecycleStart()
+{
+    id.change();
+}
+
+void
+SquidConfig::lifecycleEnd()
+{
+    // if we decide to cache an SBuf representation of lifecycles_, then we can
+    // delete that object here
```

Comment:
Let's completely remove this effectively unused method. Its current body does not make much sense outside bag61. Its caller location is somewhat controversial. We will add it later if needed.